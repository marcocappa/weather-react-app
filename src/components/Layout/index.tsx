import React from 'react';

const Layout = ({
    title,
    children,
}: {
    title: string;
    children: React.ReactNode;
}): JSX.Element => (
    <div>
        <h1>{title}</h1>
        {children}
    </div>
);

export default Layout;
