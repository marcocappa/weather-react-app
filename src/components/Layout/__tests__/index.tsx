import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../index';

describe('<Layout />', () => {
    const props = {
        title: 'title',
        children: (
            <div>
                <h2>children</h2>
                <p>other content</p>
            </div>
        ),
    };

    it('should match snapshot', () => {
        const wrapper = shallow(<Layout {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
