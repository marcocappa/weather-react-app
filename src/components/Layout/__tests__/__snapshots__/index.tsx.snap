// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should match snapshot 1`] = `
<div>
  <h1>
    title
  </h1>
  <div>
    <h2>
      children
    </h2>
    <p>
      other content
    </p>
  </div>
</div>
`;
